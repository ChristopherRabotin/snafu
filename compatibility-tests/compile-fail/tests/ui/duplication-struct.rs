use snafu::prelude::*;

#[derive(Debug, Snafu)]
enum UsableError {}

#[derive(Debug, Snafu)]
#[snafu(source(from(UsableError, Box::new)))]
#[snafu(source(from(UsableError, Box::new)))]
struct StructError(Box<UsableError>);

fn main() {}
