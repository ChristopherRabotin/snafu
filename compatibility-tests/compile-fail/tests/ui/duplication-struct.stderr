error: Multiple `source(from)` attributes are not supported on a tuple struct
 --> tests/ui/duplication-struct.rs:8:9
  |
8 | #[snafu(source(from(UsableError, Box::new)))]
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
